paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -1519,9 +1520,13 @@ class PaymentSheet internal constructor(
             )
         }
 
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+        fun onExternalPaymentMethodResult(paymentResult: PaymentResult)
```

Comment:
This is kinda weird. FlowController is a public interface that merchants can implement (idk why they would...).

But now we require them to handle this.

I'm thinking my idea in another comment could avoid this.

Reply:
This has been removed